jenkins/Jenkinsfile-external

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/groovy
+@Library('h2o3-shared-lib') _
+// Job parameters
+properties(
+        [
+                parameters(
+                        [
+                                choice(
+                                        choices: '2.2.0\n2.1.0\n2.0.2\n2.0.1\n2.0.0\n1.6.3\n1.6.2\n1.6.1\n1.6.0',
+                                        description: 'Version of Spark used for testing.',
+                                        name: 'sparkVersion'),
+
+                                booleanParam(name: 'runUnitTests', defaultValue: true, description: 'Run unit and pyunit tests'),
+                                booleanParam(name: 'runLocalIntegTests', defaultValue: true, description: 'Run local integration tests'),
+                                booleanParam(name: 'runScriptTests', defaultValue: true, description: 'Run script tests'),
+                                booleanParam(name: 'runIntegTests', defaultValue: true, description: 'Run integration tests'),
+                                booleanParam(name: 'runPySparklingIntegTests', defaultValue: true, description: 'Run pySparkling integration tests'),
+
+
+                                choice(
+                                        choices: 'yarn\nstandalone\nlocal',
+                                        description: 'Sparkling water test profile',
+                                        name: 'sparklingTestEnv'),
+
+                                choice(
+                                        choices: 'hdp2.2\nhdp2.3\nhdp2.4\nhdp2.5\nhdp2.6\ncdh5.4\ncdh5.5\ncdh5.6\ncdh5.7\ncdh5.8\ncdh5.10\nmapr4.0\nmapr5.0\nmapr5.1\nmapr5.2\niop4.2',
+                                        description: 'Hadoop version for which H2O driver is obtained',
+                                        name: 'driverHadoopVersion'),
+
+                                string(name: 'hdpVersion', defaultValue: '2.2.6.3-1', description: 'HDP version to pass to Spark configuration - for example, 2.2.0.0-2041, or 2.6.0.2.2, or current. When running external tests on yarn, the current will not do since it is not automatically expanded -> so please set 2.2.6.3-1'),
+
+                                booleanParam(name: 'buildAgainstH2OBranch', defaultValue: false, description: 'By default, Sparkling Water is built with the included H2O. This option may be used to force build to use specific H2O branch against which Sparkling Water is built.'),
+                                string(name: 'h2oBranch', defaultValue: 'master', description: 'H2O branch to build against if buildAgainstH2OBranch is set to true')
+                        ]
+                ),
+                buildDiscarder(logRotator(numToKeepStr: '10'))
+        ]
+)
+
+if (env.CHANGE_BRANCH != null && env.CHANGE_BRANCH != '') {
```

Comment:
You should use shared jenkins library to test this stuff

Reply:
@Library('test-shared-library') _

Reply:
currently this functionality is available in @Library('h2o3-shared-lib') _ . If `@library('test-shared-lib')` is on, I will change it